features/git-town-append/on-parent-branch.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Appending a branch to a parent branch
```

Comment:
I don't think this test is necessary. I can't think of a logic error where existing child branches would be modified.  The main branch is technically a parent branch as well.